bedrock/careers/templates/careers/position.html

```diff
@@ -32,25 +32,47 @@
           <dd class="job-post-team">{{ position.department }}</dd>
         {% endif %}
         <dt class="job-post-location-head">Locations:</dt>
-        <dd class="job-post-location">{{ position.job_locations }}</dd>
+        <dd class="job-post-location">{{ postings | map(attribute='job_locations') | join(', ') }}</dd>
       </dl>
     </div>
-    <div class="appy-btn">
-      <a href="{{ position.apply_url }}" class="mzp-c-button">Apply Now</a>
+    <div class="job-apply">
+    {% if postings|length == 1 %}
+      <div class="appy-btn">
```

Comment:
Safety-check: is it def meant to be `appy-btn` not `apply-btn`

Reply:
Good catch. `appy-btn` is defined in the SCSS but `apply-btn` is not. So I think this is intended.

Reply:
It definitely looks like a mistake, but a pre-existing mistake. I'm tempted to fix it anyway but it's not hurting anyone.